### ISSUE TITLE
TRS URL and TRS ID copying in the UI is odd

### ISSUE THREAD

**Issue by Richard-Hansen** [dockstore/dockstore#4688]
**Is your feature request related to a problem? Please describe.**
Perhaps this is intended, in which case this ticket can be rejected, but I found the way the UI surfaces the TRS URL and TRS ID (mainly how they are linked) a little odd.  The hyperlinks provide a URL, but the hyperlink's attached copy button provides the TRS ID. This caused some confusion when I was attempting the copy the hyperlink URL, but ended up with an ID.

For example:
<img width="584" alt="Screen Shot 2022-01-27 at 10 15 31 AM" src="https://user-images.githubusercontent.com/36607471/151387512-54bc1701-873b-4ffe-85fc-441ab50d6a95.png">

1. The hyperlink href supplies the TRS URL: https://dockstore.org/api/ga4gh/trs/v2/tools/%23workflow%2Fgithub.com%2FDataBiosphere%2Ftopmed-workflows%2FUM_variant_caller_wdl/versions/1.32.0/plain-WDL/descriptor//variant-caller/variant-caller-wdl/topmed_freeze3_calling.wdl
2. The copy icon will give you `#workflow/github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl`

**Describe the solution you'd like**
Generally, I'd say having the copy icon copy different text than the associated hyperlink is off. Perhaps we can have two icons after the hyperlink, the copy icon gives the URL and another icon (ID icon?) gives the TRS ID.

**Describe alternatives you've considered**
Ignore this feature request. It's very minor.

**Comment by denis-yuen** [comment]
Note for implementer, can be grouped with #3823

**Comment by garyluu** [comment]
yup, i also found this odd but I recall this was intended for some reason
